tests/chainer_tests/functions_tests/math_tests/test_polygamma.py

```diff
@@ -1,81 +1,60 @@
 import unittest
 
 import numpy
+import scipy
```

Comment:
You need to write this statement within the test that's decorated with `@testing.with_requires('scipy')`, as it used to be written since `scipy` is not mandatory. Otherwise, we'll get an `ImportError` when loading this test.

Reply:
yes. done.